src/frontends/ir/src/utils.cpp

```diff
@@ -98,4 +98,21 @@ bool get_dimension_from_attribute(const pugi::xml_node& node, const std::string&
     return true;
 }
 
+void str_to_set_of_strings(const std::string& value, std::set<std::string>& res) {
+    std::stringstream ss(value);
+    std::string field;
+    while (getline(ss, field, ',')) {
+        if (field.empty())
```

Comment:
looks like this condition is also checked on line 110

Reply:
you're right. line 110 condition fully covers this one.